setup.py

```diff
@@ -85,6 +92,7 @@ def run(self):
         'entrypoints',
         'tornado>=4.1',
     ],
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, <=4',
```

Comment:
We can remove `<=4`. The rest is already correctly limiting to 2.7, 3.3+. This should be:

```python
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*',
```

adding `>=3.3` would exclude 2.7, since all of these conditions are AND-ed together.

Reply:
Good point. I need more coffee.